integration/test/draining/draining_test.go

```diff
@@ -316,22 +317,26 @@ func removeWorker(clusterID string) error {
 		patch[0].Op = "remove"
 		patch[0].Path = "/spec/aws/workers/1"
 
-		err := config.Host.ApplyAWSConfigPatch(patch, clusterID)
+		patchBytes, err := json.Marshal(patch)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+		_, err = config.K8sClients.G8sClient().ProviderV1alpha1().AWSConfigs("default").Patch(clusterID, types.JSONPatchType, patchBytes)
```

Comment:
Just doing this in 3 lines now instead of using e2e-harness functions which have the `k8sClient` baked in.